textbook/15/2/pillar1.ipynb

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
<p>FYI: Indeed, it's not a big deal, but the use of&nbsp;"non-consecutive headers" upsets textbook generation. For example:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Pillar 1
### The attainment, usage, storage, analysis, and maintenance…
</pre><p> </p><p>These headers are used for declaring sections within each document, after all. Perhaps it would make sense to merely set that line to be bold:</p><p> </p><pre class="ql-syntax" spellcheck="false">**The attainment, usage, storage, analysis, and maintenance…**
</pre><p> </p><p>…And the same with others in this chapter:</p><p> </p><pre class="ql-syntax" spellcheck="false">textbook/15/2/pillar1.ipynb:10003: WARNING: Non-consecutive header level increase; 1 to 3 [myst.header]
textbook/15/3/pillar2.ipynb:10003: WARNING: Non-consecutive header level increase; 1 to 3 [myst.header]
textbook/15/4/pillar3.ipynb:10003: WARNING: Non-consecutive header level increase; 1 to 3 [myst.header]
textbook/15/5/pillar4.ipynb:10003: WARNING: Non-consecutive header level increase; 1 to 3 [myst.header]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
And to be sure, this is how it looks in the textbook:

![image](https://user-images.githubusercontent.com/530998/206009934-6ac0f779-6970-4e25-a1f9-85038669de3b.png)